### ISSUE TITLE
Bad global javascript config crashes webapp

### ISSUE THREAD

**Issue by bennlich** [jhpyle/docassemble#165]
Seeing the following error in `/var/log/apache2/error.log` whenever I try to load the docassemble webapp in my browser (e.g. https://mycitations.dev.courts.ca.gov/):

```
mod_wsgi (pid=85074): Target WSGI script '/usr/share/docassemble/webapp/docassemble.wsgi' cannot be loaded as Python module.
mod_wsgi (pid=85074): Exception occurred processing WSGI script '/usr/share/docassemble/webapp/docassemble.wsgi'.
Traceback (most recent call last):
  File "/usr/share/docassemble/webapp/docassemble.wsgi", line 1, in <module>
    from docassemble.webapp.server import app as application
  File "/usr/share/docassemble/local3.5/lib/python3.5/site-packages/docassemble/webapp/server.py", line 716, in <module>
    user_manager.init_app(app, db_adapter=db_adapter, login_form=MySignInForm, register_form=MyRegisterForm, user_profile_view_function=user_profile_page, logout_view_function=logout, unauthorized_view_function=unauthorized, unauthenticated_view_function=unauthenticated, login_view_function=custom_login, register_view_function=custom_register, resend_confirm_email_view_function=custom_resend_confirm_email, resend_confirm_email_form=MyResendConfirmEmailForm, password_validator=password_validator)
  File "/usr/share/docassemble/local3.5/lib/python3.5/site-packages/flask_user/__init__.py", line 177, in init_app
    app.register_blueprint(blueprint)
  File "/usr/share/docassemble/local3.5/lib/python3.5/site-packages/flask/app.py", line 66, in wrapper_func
    return f(self, *args, **kwargs)
  File "/usr/share/docassemble/local3.5/lib/python3.5/site-packages/flask/app.py", line 1107, in register_blueprint
    blueprint, self.blueprints[blueprint.name], blueprint.name
AssertionError: A name collision occurred between blueprints <flask.blueprints.Blueprint object at 0x7f94fd33b320> and <flask.blueprints.Blueprint object at 0x7f95112dac88>. Both share the same name "flask_user". Blueprints that are created on the fly need unique names.
```
Does this mean anything obvious to you?

@JCCChris tells me this started after adding the following line to config.yml via the /config interface and restarting:
```
global javascript: docassemble.jcc.abilitytopay:data/static/nodep-date-input-polyfill.dist.js
```
though I'm skeptical that that line is actually the culprit. Seems more likely that something went wrong during app restart?

I think we're running DA version 0.4.53, though I'm not positive where in the container to get this info. (I usually check the web interface.)

**Comment by jhpyle** [comment]
You need to look back further in the error.log to find the true error.

See https://docassemble.org/docs/docker.html#troubleshooting

> If you are get a “server error” in your web browser when trying to access docassemble, there should be an error message in /var/log/apache2/error.log. If you see a message about a “blueprint’s name collision,” this is almost always not the real error; you need to scroll up through several error messages to find the actual error. When Apache crashes, the error that initiated the crash causes other errors inside of the code of the Flask framework, and a “blueprint’s name collision” error is typically the last error to be recorded in the error log.

**Comment by bennlich** [comment]
Thanks. Here's the real error:

```
mod_wsgi (pid=85074): Target WSGI script '/usr/share/docassemble/webapp/docassemble.wsgi' cannot be loaded as Python module.
mod_wsgi (pid=85074): Exception occurred processing WSGI script '/usr/share/docassemble/webapp/docassemble.wsgi'.
Traceback (most recent call last):
  File "/usr/share/docassemble/webapp/docassemble.wsgi", line 1, in <module>
    from docassemble.webapp.server import app as application
  File "/usr/share/docassemble/local3.5/lib/python3.5/site-packages/docassemble/webapp/server.py", line 21492, in <module>
    global_js += "\\n" + '    <script src="' + get_url_from_file_reference(fileref) + '"></script>';
TypeError: Can't convert 'NoneType' object to str implicitly
```

**Comment by jhpyle** [comment]
Thanks.  This should be fixed in 0.4.59.
From the command line after a `docker exec`, you can do:
```
su www-data
source /usr/share/docassemble/local3.5/bin/activate
pip install --upgrade docassemble.webapp docassemble docassemble.base docassemble.demo
exit
```

**Comment by bennlich** [comment]
:+1: works! thank you!
